pensieve/experimenter.py

```diff
@@ -0,0 +1,48 @@
+import datetime as dt
+from typing import List, Optional
+
+import attr
+import cattr
+import requests
+import pytz
+
+
+@attr.s
+class Variant:
+    is_control: bool = attr.ib()
+    slug: bool = attr.ib()
+    ratio: int = attr.ib()
+
+
+@attr.s
+class Experiment:
+    slug: str = attr.ib()
+    start_date: Optional[dt.datetime] = attr.ib()
+    end_date: Optional[dt.datetime] = attr.ib()
+    variants: List[Variant] = attr.ib()
+
+
+@attr.s
+class ExperimentCollection:
+    experiments: List[Experiment] = attr.ib(default=attr.Factory(list))
+
+    EXPERIMENTER_API_URL = "https://experimenter.services.mozilla.com/api/v1/experiments/"
+
+    @staticmethod
+    def _num_to_datetime(obj, dest_type) -> dt.datetime:
+        if obj is None:
+            return None
+        return dt.datetime.fromtimestamp(obj / 1e6, pytz.utc)
+
+    @classmethod
+    def from_experimenter(cls, session: requests.Session = None) -> "ExperimentCollection":
+        session = session or requests.Session()
+        experiments = session.get(cls.EXPERIMENTER_API_URL).json()
+        converter = cattr.Converter()
+        converter.register_structure_hook(dt.datetime, cls._num_to_datetime)
+        return cls([converter.structure(experiment, Experiment) for experiment in experiments])
+
+    def started_since(self, since: dt.datetime) -> "ExperimentCollection":
```

Comment:
Is exercising this something you're planning to add in future tests? Right now it looks unused.

Reply:
Added a test and fixed a bug.